packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { ICalendarDayGridStyleProps, ICalendarDayGridStyles } from './CalendarDayGrid.types';
+import { FontSizes, FontWeights, getFocusStyle, getGlobalClassNames } from '@uifabric/styling';
+import { DateRangeType } from 'office-ui-fabric-react/lib/utilities/dateValues/DateValues';
+
+const GlobalClassNames = {
+  hoverStyle: 'ms-CalendarDay-hoverStyle',
+  pressedStyle: 'ms-CalendarDay-pressedStyle'
+};
+
+export const styles = (props: ICalendarDayGridStyleProps): ICalendarDayGridStyles => {
+  const { theme, dateRangeType, showWeekNumbers, lightenDaysOutsideNavigatedMonth } = props;
+  const { palette } = theme;
+
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const disabledStyle = {
+    selectors: {
+      '&, &:disabled, & button': {
+        color: palette.neutralTertiaryAlt,
+        pointerEvents: 'none'
+      }
+    }
+  };
+
+  return {
+    table: [
+      {
+        textAlign: 'center',
+        borderCollapse: 'collapse',
+        borderSpacing: '0',
+        tableLayout: 'fixed',
+        fontSize: 'inherit',
+        marginTop: 4,
+        width: 197
+      },
+      showWeekNumbers && {
+        width: 226
+      }
+    ],
+    dayCell: {
+      margin: 0,
+      padding: 0,
+      width: 28,
+      height: 28,
+      lineHeight: 28,
+      fontSize: FontSizes.small,
+      fontWeight: FontWeights.regular,
+      color: palette.neutralPrimary,
+      cursor: 'pointer',
```

Comment:
Week days ('S', 'M', etc.) also adopt this class which causes cursor change on hover even though they aren't interactive. Since dayButton class has the cursor change, could this be deleted?

Reply:
not quite that simple since dayButton is the inner cell (so it doesn't cover the edge of the squares between the buttons). Since I'd have to rework the classes to fix this one I'll probably leave it for next time